### ISSUE TITLE
Extension does not load and no errors shown.

### ISSUE THREAD

**Issue by dgtllion** [idahogurl/vs-code-prettier-eslint#254]
# GitHub Bug Report

**Describe the bug**
Prettier formatting does not work in Cursor despite having the proper configuration in settings.json. When attempting to format code (either manually or on save), no formatting changes occur in any file type.

**To Reproduce**
1. Install Cursor editor
2. Configure settings.json with Prettier settings (both with official prettier and prettier-eslint)
3. Open a JavaScript/TypeScript/JSON file with formatting issues
4. Try to format the document using keyboard shortcut (Shift+Alt+F or equivalent)
5. Alternatively, save the file with formatOnSave enabled
6. Observe that no formatting takes place

**Expected behavior**
Code should be automatically formatted according to Prettier rules when saving a file or manually triggering formatting.

**Example Project**
N/A

**Screenshots**
N/A

**Versions (please complete the following information):**
- VS Code Prettier ESLint extension: 3.1.0
- Cursor (VS Code fork): Version: 0.47.8 (Universal)
  - VSCode Version: 1.96.2
- Node: 22.14.0
- Package Manager: npm 10.9.2
- prettier@^3.1.0
- eslint@^8.52.0
- prettier-eslint@^16.1.2

**System Specifications (please complete the following information):**
- OS: macOS
- Processor: M1
- RAM Size: 16GB

**Additional context**
I've tried multiple approaches to fix this issue:

1. Using the standard Prettier extension (esbenp.prettier-vscode) (**this works!**)
2. Using the Prettier ESLint extension (rvest.vs-code-prettier-eslint) (NOT WORKING! Also no errors in output.)
3. Different configuration settings in settings.json including:
   - editor.formatOnSave: true
   - editor.defaultFormatter set for specific languages
   - editor.formatOnSaveMode: "file"
   - Various ESLint and Prettier specific settings

My current settings.json file:
```json
{
  "window.commandCenter": true,
  "javascript.format.enable": false,
  "editor.codeActionsOnSave": {
    "source.formatDocument": "always",
    "source.fixAll": "explicit",
    "source.fixAll.eslint": "explicit"
  },
  "terminal.integrated.commandsToSkipShell": [],
  "editor.tabSize": 2,
  "workbench.colorCustomizations": {
    "tab.activeBorder": "#ff0000",
    "tab.unfocusedActiveBorder": "#FFCC00"
  },

  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
  "editor.formatOnType": true,
  "editor.formatOnPaste": true,
  "editor.formatOnSave": true,
  "editor.formatOnSaveMode": "file",
  "files.autoSave": "onFocusChange",
  "vs-code-prettier-eslint.prettierLast": false
}
```

```json
// .prettierrc
{
  "trailingComma": "all",
  "tabWidth": 2,
  "printWidth": 120,
  "semi": true,
  "singleQuote": true,
  "bracketSpacing": true
}
```

```json
//.eslintrc.json
{
  "extends": ["next/core-web-vitals", "airbnb", "airbnb-typescript", "prettier"],
  "root": true,
  "env": {
    "es2021": true
  },
  "globals": {
    "React": "readonly"
  },
  "parserOptions": {
    "project": "./tsconfig.json"
  },
  "rules": {
    "no-unused-vars": "off",
    "no-underscore-dangle": [
      "error",
      {
        "allow": ["_id"]
      }
    ],
    "@typescript-eslint/no-unused-vars": [
      "error",
      {
        "argsIgnorePattern": "^_"
      }
    ],
    "react/prop-types": "off",
    "no-nested-ternary": "off",
    "react/jsx-props-no-spreading": "off",
    "import/prefer-default-export": "off",
    "no-plusplus": [
      "error",
      {
        "allowForLoopAfterthoughts": true
      }
    ],
    "@typescript-eslint/indent": "off",
    "react/no-unknown-property": [
      "error",
      {
        "ignore": ["fill", "jsx", "global"]
      }
    ],
    "react/require-default-props": "off",
    "jsx-a11y/anchor-is-valid": [
      "error",
      {
        "components": ["Link"],
        "specialLink": ["hrefLeft", "hrefRight"],
        "aspects": ["invalidHref", "preferButton"]
      }
    ]
  }
}

```

Formatting still does not work despite numerous configuration attempts. The issue appears to be with Cursor's handling of formatting extensions rather than with the configuration itself.
